infrastructure/builder/src/utils/config.js

```diff
@@ -48,14 +48,15 @@ exports.configToSchema = type => {
 /**
  * Convert lib-config !env types into example values
  */
-exports.configToExample = type => {
+exports.configToExample = (type, optional) => {
+  let expl = optional ? '(optional) ' : '';
   switch (type) {
-    case '!env': return '...';
-    case '!env:string': return '...';
-    case '!env:number': return '...';
-    case '!env:bool': return 'true/false';
-    case '!env:json': return {};
-    case '!env:list': return [];
+    case '!env': return `${expl}...`;
+    case '!env:string': return `${expl}...`;
+    case '!env:number': return `${expl}...`;
+    case '!env:bool': return `${expl}true/false`;
+    case '!env:json': return `${expl}{}`;
```

Comment:
this ends up not being valid json.. (`foo: '(optional) {}'`)

Reply:
Yeah, I've given up on it being valid and just being strings that say what should be there.

Reply:
Oh wait shoot, this gets pasted into dev-configs too. ugh